source/Coop.Core/Server/Connections/Messages/NetworkModuleVersionsValidate.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Common.Logging;
+using Common.Messaging;
+using GameInterface.Services.Modules;
+using ProtoBuf;
+using Serilog;
+
+namespace Coop.Core.Server.Connections.Messages;
+
+/// <summary>
+/// Message from Client to Server for validating the module versions.
+/// Responsibilities
+/// 1. Make sure that all active modules have the same version as on the server
+/// </summary>
+[ProtoContract(SkipConstructor = true)]
+public record NetworkModuleVersionsValidate : ICommand
+{
+    [ProtoMember(1)]
+    public List<NetworkModuleInfo> Modules { get; }
```

Comment:
Change this to an array as it is easier to serialize and takes less bandwidth (kind of pedantic but I noticed this)